.travis.yml

```diff
@@ -5,8 +5,17 @@ python:
 install:
   - pip install -r requirements.txt
 
+services:
+  - postgresql 
+  
+install:
+  - pip install -r requirements.txt
+
+before_script:
+  - psql -c 'create database covidtest;' -U postgres
+
 script:
-  - "python -m unittest test_reviews_model.py"
-  - "python -m unittest test_reviews_views.py"
-  - "python -m unittest test_user_model.py"
-  - "python -m unittest test_user_views.py"
+  - "python -m unittest tests/test_reviews_model.py"
+  - "python -m unittest tests/test_reviews_views.py"
+  - "python -m unittest tests/test_user_model.py"
+  - "python -m unittest tests/test_user_views.py"
```

Comment:
@azaria-dedmon please refactor accordingly so that only have one script running all tests. In the event that we have 500 tests files, the codebase can become hard to maintain. Consider discovering all tests in the tests folder i.e `python -m unittest discover`. Refer to this resource ---> [https://docs.python.org/3/library/unittest.html#test-discovery](https://docs.python.org/3/library/unittest.html#test-discovery)